components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -2511,7 +2515,7 @@ def define(self):
 static NAMESPACE_OBJECT_CLASS: NamespaceObjectClass = unsafe {
     NamespaceObjectClass::new(%s)
 };
-""" % str_to_const_array(classString)
+""" % str_to_byte_slice(classString)
```

Comment:
Here we can change `NamespaceObjectClass::new` to accept CStr: https://doc.servo.org/src/script/dom/bindings/namespace.rs.html#25-34